tests/tasks/task_bodies.py

```diff
@@ -1,10 +1,11 @@
 """TES task request bodies used by tests"""
 
 from pathlib import Path
+from tempfile import TemporaryDirectory
 
 DIR = Path(__file__).parents[2]
 input_dir = DIR / "inputs"
-output_dir = DIR / "outputs"
+tmp_dir = TemporaryDirectory()  # pylint: disable=consider-using-with
```

Comment:
I think it is better to manage state at the local, rather than at the global scope. That is, if you make `tmp_dir` a global variable you essentially have no control over when the temporary directory is created and deleted. I would suggest making the task bodies into functions, and then either passing the temporary directory as an argument or creating it in the body of the function. That is:
```python
def get_uppercase_task_body(tmp_dir):
   return {..., "url": f"file://{tmp_dir.name}/hello-upper.txt", ...}

```

Reply:
I ended up making the temporary directory a fixture. I wanted to use pytest's `tmp_path` but the directory's permissions don't allow funnel to write to the files in it, so the `output_dir` fixture basically serves that purpose.